docs/getting-started/token-summary/token-migration/index.md

```diff
@@ -0,0 +1,62 @@
+---
+id: cfg-migration
+title: Legacy CFG/WCFG Migration to Ethereum
+---
+
+# Legacy CFG/WCFG Migration to Ethereum
+
+With the launch of Centrifuge V3, a multi-chain, EVM-based protocol, the legacy Centrifuge Chain and its associated CFG token will be deprecated, along with the Wrapped CFG (WCFG) token on Ethereum. This guide outlines how users can migrate their legacy CFG and WCFG tokens to the new EVM-based CFG token on Ethereum.
+
+The migration process is set to begin in **April 2025**, after a successful [governance vote](https://gov.centrifuge.io/t/cp149-migration-of-centrifuge-governance-token-cfg-to-evm/6810). The existing Centrifuge Chain is expected to be discontinued in **Q4 2025**. Detailed instructions and demo videos are provided below to assist token holders.
+
+## Migration Overview
+
+- **Token Swap Ratio**: 1:1 (1 legacy CFG or WCFG = 1 new CFG)
+- **New CFG Token Address on Ethereum**: 0xcccCCCcCCC33D538DBC2EE4fEab0a7A1FF4e8A94
+- **WCFG/CFG Swap Contract on Ethereum**: TBA
+- **Total Supply**: 675'000'000M
+- **Start Date**: April X 2025
+
+## How to Migrate Your Tokens
+
+### Use the Centrifuge App
+
+#### Legacy CFG (CFG on Centrifuge Chain)
+
+1. Navigate to [https://app.centrifuge.io/migrate](https://app.centrifuge.io/migrate).
+2. Connect your wallet that holds legacy CFG.
+3. Follow the on-screen instructions:
+   - Connectan EVM-based address and sign message proving owernship of your Ethereum wallet. 
+   - Submit a transaction on the Centrifuge Chain to request migration.
+4. Upon completion, the new CFG tokens will be sent to your EVM wallet.
```

Comment:
@0x4Graham Should we add a section about payed fees here, if not I would definitely add a field in the UI that is showing them the amount of fees they will pay and then the amount the will receive on Ethereum.